sdk/containerregistry/Azure.ResourceManager.ContainerRegistry/src/Custom/ContainerRegistryCredentials.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.ResourceManager.ContainerRegistry.Models
+{
+    /// <summary> The parameters that describes a set of credentials that will be used when a run is invoked. </summary>
+    public partial class ContainerRegistryCredentials
+    {
+        /// <summary>
+        /// The authentication mode which determines the source registry login scope. The credentials for the source registry
+        /// will be generated using the given scope. These credentials will be used to login to
+        /// the source registry during the run.
+        /// </summary>
+        [WirePath("sourceRegistry.loginMode")]
+        public SourceRegistryLoginMode? SourceRegistryLoginMode
```

Comment:
Please also add a `[EditorBrowsable(EditorBrowsableState.Never)]` attribute to this property.
Since in your new library, the class where this property comes from has more than 1 property, therefore we should discourage new users to use this flattened property. Adding the attribute would hide it from intellisense.

Reply:
Made requested changes